cli/src/main.rs

```diff
@@ -277,8 +277,8 @@ fn run<I: IntoIterator<Item = T>, T: Into<OsString> + Clone>(args: I) -> Result<
                                 .long("rate")
                                 .value_name("RATE")
                                 .long_help(
-                                    "The number of batches per second to submit to the target, \
-                                defaults to 1",
+                                    "Time between batch submissions, \
```

Comment:
Same as the man page this should explain the format for time intervals and that it takes floats as well.

Reply:
expanded explanation